backends/platform/sdl/sdl.h

```diff
@@ -188,6 +189,9 @@ class OSystem_SDL : public ModularMixerBackend, public ModularGraphicsBackend {
 	bool setGraphicsMode(int mode, uint flags) override;
 	int getGraphicsMode() const override;
 #endif
+
+	bool _haveDoubleClickTime;
+	uint32 _doubleClickTime;
```

Comment:
I wonder if we need the bool - would (_doubleClickTime != 0) suffice where needed?

Reply:
I was going to suggest the same